File: single-language-analysis/dataset-files/python/test_smell_162.py
# Original URL: https://github.com/Phlogistique/qibuild/blob/27876e99deac05f60984468ec4eb48ba693dbc2c/python/qipkg/test/test_qipkg.py#L177-L179

def test_smell_162(qipkg_action):
    pkg_path = os.path.join(os.path.dirname(__file__), "projects", "python_services.pkg")
    qipkg_action("validate_package", pkg_path)

File: single-language-analysis/dataset-files/java/TestSmell162.java
import org.junit.Test;
import static org.junit.Assert.*;
import java.io.File;

public class TestSmell162 {

    @Test
    public void testValidatePackage() {
        String pkgPath = new File("projects/python_services.pkg").getAbsolutePath();
        qipkgAction("validate_package", pkgPath);
    }
}
